### PR TITLE
Removed compilator warnings

### PR DESCRIPTION
**Pull request by sisco0**
Compilator warnings were removed, same as shown at issue #107

### CODE REVIEW

**Comment by sisco0** [comment]
Have a good day, changes were accomplished in order to solve some compilation warnings.

File: src/feature/gaussian.hh [hunk 1]
```diff
@@ -9,101 +9,116 @@
 #include "lib/utils.hh"
 #include "lib/timer.hh"
 #include "common/common.hh"
-namespace pano {
+namespace pano
+{
```
Comment (ppwwyyxx) [inline src/feature/gaussian.hh:13]: most of this PR are not related to compiler warnings

Comment (sisco0) [reply src/feature/gaussian.hh:13]: Have a good day, automatic re-indentation process was accomplished by my IDE, I am disabling it in order to change the correct lines only.

File: src/lib/geometry.hh [hunk 2]
```diff
@@ -11,260 +11,400 @@
...
-		}
+template <typename T>
+class Vector
+{
+public:
+	T x = 0, y = 0, z = 0;
+
+	constexpr explicit Vector(T m_x = 0, T m_y = 0, T m_z = 0) : x(m_x), y(m_y), z(m_z) {}
+
+	constexpr Vector(const Vector<T> &rhs) = default;
```
Comment (sisco0) [inline src/lib/geometry.hh:45]: Change to keep

File: src/feature/gaussian.hh [hunk 3]
```diff
@@ -9,101 +9,116 @@
...
 			}
 
 			// apply to rows
-			REP(i, h) {
+			REP(i, h)
+			{
 				T *dest = ret.ptr(i);
-				memcpy(cur_line, dest, sizeof(T) * w);
-				{	// pad the border
+				memcpy((void *)cur_line, (void *)dest, sizeof(T) * w);
```
Comment (sisco0) [inline src/feature/gaussian.hh:82]: Change to keep

File: src/feature/gaussian.hh (hunk 1 above)
Comment (sisco0) [reply src/feature/gaussian.hh:13]: Comments added on really helpful changes.

**Comment by sisco0** [comment]
Have a good day, fixes have been applied without re-indentation of the code.

**Comment by ppwwyyxx** [comment]
thanks!
